pkg/users/users_api.go

```diff
@@ -14,7 +14,8 @@ func (a *Users) GetUser() (*User, error) {
 	return &resp.Actor.User, nil
 }
 
-const getUserQuery = `query { actor { user {
```

Comment:
Are you using the latest version of tutone? This should not have changed 🤔

Reply:
Should be good now.